org.rdkit.knime.nodes/src/org/rdkit/knime/nodes/addconformers/RDKitAddConformersNodeDialog.java

```diff
@@ -81,6 +83,8 @@
 	// Constants
 	//
 
+	public static final EmbedParameters rdkit_defaultParameters = RDKFuncs.getETKDGv3();

```

Comment:
When embedding defaults I would suggest to replace also the existing default definitions with values coming directly from RDKit. See all DEFAULT_XXX values. I would mark them all as deprecated (to be removed soon), and assign for now the resp. values from rdkit_defaultParameters. 
BTW, constants (defined as "static final") are written in all upper case: RDKIT_DEFAULT_PARAMETERS.

Reply:
makes sense